bin/regenerate_sqlx_data.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+################################################################################
+# Our Rust code uses a library called `sqlx` to interface with Postgres.
+# It *statically* checks the validity of queries by running them against a real,
+# migrated SQL database and storing that validity in sqlx-data.json.
+#
+# Long story short: Changing Rust sql queries? You probably want to run this
+# script with `make regenerate-sqlx-data`.
+################################################################################
```

Comment:
@andrea-grapl Do you happen to have that script that we came up with that does this? As I recall, I think we were able to get away with something a little simpler?